app/views/items/show.html.erb

```diff
@@ -14,7 +14,7 @@
         No current bids
       </p>
       <% else %>
-        Highest Bid: <%= @item.high_bid %><br>
+        Highest Bid: <%= number_to_currency(@item.high_bid) %><br>
```

Comment:
nice helper. does the method definition show up in another commit? or is it already on dev?